index.html

```diff
@@ -91,16 +98,15 @@ <h3>Multi-Post Stories</h3>
             <li><button type="button">Ruby</button></li>
           </ul>
         </div>
-        <button class="see-works cards-btn" type="button">See Project</button>
-      </div>
+        <button class="see-works cards-btn" type="button"  id="multipost-btn">See Project</button>
```

Comment:
- Nice work you have done with your project to make sure everything looks good with the card kindly move the button to the bottom of the card, so it aligns with the Figma Template.

## Figma Design
![card](https://user-images.githubusercontent.com/48371156/166103741-1a09e29f-9945-4aba-85fb-8f32a25a1f35.png)

## Your Design
![card2](https://user-images.githubusercontent.com/48371156/166103750-0c00dc95-8319-4f9e-ac2f-3275db57013c.png)